### ISSUE TITLE
Paper wallet creation date format is ambiguous

### ISSUE THREAD

**Issue by lycurgus** [urbit/bridge#529]
**Describe the bug**
Creation date of paper wallet PNGs is shown in an ambiguous format.

**To Reproduce**
1. Create a paper wallet e.g. via reticketing
2. Observe that the creation date is shown in dd/mm/yy format

**Expected behavior**
It would be nice if the date could be shown in ~yy.mm.dd (or ideally ~yyyy.mm.dd)

**Screenshots**
![image](https://user-images.githubusercontent.com/8443742/95733128-69ee8380-0ccd-11eb-9fe9-b8ed14e035cb.png)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox 81

**Login method (please complete the following information):**
Metamask (though I assume it's the same if you generate a wallet when already logged in via Master Ticket...)

**Comment by urcades** [comment]
Yeah, this is a good suggestion! Seems simple enough to switch around in the wallet renderer.

**Comment by lycurgus** [comment]
Took me a little while but I figured out where the wallet rendering was happening and have now raised urbit/PaperRenderer/pull/156.
